website/content/docs/connect/gateways/api-gateway/configuration/routetimeoutfilter.mdx

```diff
@@ -97,18 +97,18 @@ Map that contains the details about the gateway policy. The `apiVersion`, `kind`
 
 ### `spec.idleTimeout
 
-Specifies the total amount of time permitted for the request stream to be idle.
+Specifies the total amount of time permitted for the request stream to be idle. Must specify a parseable number and a unit. Example: "5s" 
```

Comment:
```suggestion
Specifies the total amount of time permitted for the request stream to be idle. Must specify a parseable number and a unit, for example "5s". 
```
If the default value is an empty string, what is the behavior? Does that mean there is no timeout and the stream can remain idle indefinitely?

Reply:
I believe if its not set it defaults to whatever envoy defaults to when not specified, which according to the envoy docs appears to be 1 hour https://www.envoyproxy.io/docs/envoy/latest/api-v3/config/core/v3/protocol.proto#envoy-v3-api-field-config-core-v3-httpprotocoloptions-idle-timeout.